lib/Dialect/OM/Evaluator/Evaluator.cpp

```diff
@@ -120,6 +121,37 @@ circt::om::Evaluator::instantiate(
   return success(std::shared_ptr<evaluator::ObjectValue>(object));
 }
 
+bool hasOMTarget(StringRef path) {
+  return (path.startswith("OMDontTouchedReferenceTarget") ||
+          path.startswith("OMMemberInstanceTarget") ||
+          path.startswith("OMMemberReferenceTarget") ||
+          path.startswith("OMReferenceTarget"));
```

Comment:
+ `OMInstanceTarget` (Chisel emits, added to CIRCT after this PR I believe).

This already exists, but gosh it'd be nice if every pass touching this concept didn't have its own local list of the "recognized" target kinds.